CMakeLists.txt

```diff
@@ -1,70 +1,96 @@
-cmake_minimum_required (VERSION 3.13)
-project (brainflow)
```

Comment:
these code style changes are not related to pieeg, and should be reverted

Reply:
Reverted the code style changes not related to PIEEG as requested.